packages/components/input/stories/input.stories.tsx

```diff
@@ -90,7 +90,12 @@ const PasswordTemplate: ComponentStory<typeof Input> = (args) => {
       <Input
         {...args}
         endContent={
-          <button className="focus:outline-none" type="button" onClick={togglePasswordVisibility}>
+          <button
+            aria-label="show password"
+            aria-pressed={isPasswordVisible}
```

Comment:
This param here is not needed since this is a native button

Reply:
Native buttons do not have a toggle state, this would only not be needed if it would for example be a checkbox.